robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowView.java

```diff
@@ -133,7 +134,7 @@ protected Drawable buildDrawable(int resourceId) {
   }
 
   protected String getQualifiers() {
-    return shadowOf(realView.getResources().getConfiguration()).getQualifiers();
```

Comment:
Maybe we want to keep this lookup happening without going through `RuntimeEnvironment` because injection over statics? Seems possible that views could have different origin Contexts and Resources, especially in a multi-app case?